notebooks/plots/text.ipynb

```diff
@@ -84,24 +188,11 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
```

Comment:
some cells are run and some are not, this is confusing

Reply:
Not meant to update this notebook so will clear output.